source/github-helpers/api.ts

```diff
@@ -209,6 +209,45 @@ export const v4 = mem(async (
 	cacheKey: JSON.stringify,
 });
 
+export const v4mutation = mem(async (
```

Comment:
Mutations should not be memoized. But also this specific method doesn't look necessary, why did you create it?

```suggestion
export const v4mutation = async (
```

Reply:
But actually I think it's best to merge this into the regular v4, perhaps by detecting and allowing the starting `mutation` keyword

Reply:
Merged the two functions